corehq/apps/app_manager/ui_translations/ui_translations.py

```diff
@@ -128,7 +128,7 @@ def build_ui_translation_download_file(app):
 
     rows = list(row_dict.values())
     try:
-        commcare_version = str(StrictVersion(app.build_version.vstring))
+        commcare_version = str(LooseVersion(app.build_version.vstring))
```

Comment:
What's the difference between `StrictVersion` and `LooseVersion`? The names make it sound like they are quite different. Is this change safe?

Reply:
based on the tests and reviewing how we use the build numbers in HQ, I believe this change is safe. It's just determining the correct build number from the app string.

Reply:
great question! looking around, there isn't a direct test for `build_ui_translation_download_file` but it looks like when @dannyroberts [made this change](https://github.com/dimagi/commcare-hq/commit/c045b31046ffb132fb98a1d088b0e101cf7dacb3) there appears to be a [test related to this change](https://github.com/dimagi/commcare-hq/commit/c045b31046ffb132fb98a1d088b0e101cf7dacb3#diff-3b022e634e4f233d3afdfa35cf7831ac68e5858beaca8cdea196f3a27ab71ddbR21)

@dannyroberts I know this was quite a while ago, but do you recall why we moved to use `StrictVersion` here over `LooseVersion`? Thanks!